draft-ietf-openpgp-pqc.md

```diff
@@ -885,6 +891,14 @@ The algorithm ID identifies unequivocally the algorithm, the parameters for its
 This specification makes use of the default "hedged" variants of ML-DSA and SLH-DSA, which mix fresh randomness into the respective signature-generation algorithm's internal hashing step.
 This has the advantage of an enhanced side-channel resistance of the signature operations according to  {{FIPS-204}} and {{FIPS-205}}.
 
+## Minimum digest size for PQ(/T)-signatures
+
+This specification requires that all PQ(/T) signatures defined in this document are made in combination with a hash algorithm of at least 256 bits digest size.
+The relevant security property is preimage resistance, since all signature algorithms defined in this document require signature packets of version 6 or higher, which include a random salt value.
```

Comment:
```suggestion
The relevant security property is preimage resistance, since all signature algorithms defined in this document require signature packets of version 6 or higher, which currently include a random salt prefix.
```